python/feathub/feature_views/feature_view.py

```diff
@@ -136,18 +150,17 @@ def get_output_fields(self, source_fields: List[str]) -> List[str]:
 
         return list(OrderedDict.fromkeys(reordered_output_fields))
 
-    def get_feature(self, feature_name: str) -> Feature:
+    def get_output_features(self) -> List[Feature]:
         if self.is_unresolved():
             raise RuntimeError("Build this feature view before getting features.")
-        for feature in self.get_resolved_features():
-            if feature_name == feature.name:
-                return feature
 
-        if not self.keep_source_fields:
-            raise RuntimeError(
-                f"Failed to find the feature '{feature_name}' in {self.to_json()}."
+        output_features = {}
+        if self.keep_source_fields:
+            output_features.update(
+                {f.name: f for f in self.get_resolved_source().get_output_features()}
             )
-        return cast(TableDescriptor, self.source).get_feature(feature_name)
+        output_features.update({f.name: f for f in self.get_resolved_features()})
+        return list(output_features.values())
```

Comment:
Would it be useful to make the order of the features in this list the same as the order of features outputted by the FeatureView, so that we can reuse this list to construct the FeatureView's schema?